beacon-chain/p2p/broadcaster.go

```diff
@@ -100,6 +100,8 @@ func (s *Service) BroadcastSyncCommitteeMessage(ctx context.Context, subnet uint
 }
 
 func (s *Service) internalBroadcastAttestation(ctx context.Context, subnet uint64, att ethpb.Att, forkDigest [fieldparams.VersionLength]byte) {
+	const minimumPeersPerSubnet = 1
```

Comment:
hmm this shouldn't be some kind of field parameter or config parameter? feels weird that it's just some constant in a function